### PR TITLE
Contribution: English to Turkish Translations

### PR DESCRIPTION
**Pull request by devosonder**

### CODE REVIEW

**Comment by gitforkgit** [comment]
First of all, wow. Amazing how much you translated :o. To merge this big pull requests I would suggest that if you contribute something else you always add a description of what exactly you added. It may take some time to merge this pull request because it is big. I do my best. :+1:

**Comment by devosonder** [comment]
> 25 files reviewed yet.

Oh my God. That's too few. Is there a reason it's taking so long?

**Comment by gitforkgit** [comment]
Not really, I have now taken time for it :+1:

File: Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json [hunk 1]
```diff
@@ -210,12 +291,12 @@
                     "text": "sandBox bazı borsalarda, tüccarların simüle edilmiş bir ortamda tüm borsa özelliklerini denemelerini sağlayan testnet/sandBox modu vardır. Etkinleştirmek için true olarak ayarlayın. Lütfen bazı borsaların (örneğin Bittrex gibi) bu parametrenin false olarak ayarlanmasıyla hata verebileceğini unutmayın. Bu gibi durumlarda o satırı silmeniz yeterlidir.",
                     "updated": 1652799269237
                 }
-            ],
-            "updated": 1650635009708
+            ]
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json:294]: Did you make those deletions by hand or did they happen by accident ?

File: Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+                "text": "Data Tasks относятся к интеллектуальному анализу данных, т.е. к деятельности по обработке данных. Вам необходимо обрабатывать данные для подачи графиков, чтобы Ваши торговые системы могли принимать решения на основе качественной информации.",
+                "updated": 1638356535434
+            },
+            {
+                "language": "TR",
+                "text": "Veri görevleri, veri madenciliğini, yani veri işleme faaliyetini ifade eder. Grafikleri beslemek için verileri işlemeniz gerekir ve böylece ticaret sistemleriniz kaliteli bilgilere dayalı kararlar verebilir.",
+                "updated": 1639120026724
+            }
+        ]
+    },
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json:24]: Did you translate into all this languages or did this happen by accident?

File: Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json [hunk 3]
```diff
@@ -65,35 +105,27 @@
...
-                "text": "Data Tasks относятся к интеллектуальному анализу данных, т.е. к деятельности по обработке данных. Вам необходимо обрабатывать данные для подачи графиков, чтобы Ваши торговые системы могли принимать решения на основе качественной информации.",
-                "updated": 1638356535434
-            },
-            {
-                "language": "TR",
-                "text": "Veri görevleri, veri madenciliğini, yani veri işleme faaliyetini ifade eder. Grafikleri beslemek için verileri işlemeniz gerekir ve böylece ticaret sistemleriniz kaliteli bilgilere dayalı kararlar verebilir.",
-                "updated": 1639120026724
-            }
-        ]
-    }
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json:98]: Is this related to the translations added above?

File: Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json (hunk 1 above)
Comment (devosonder) [reply Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json:294]: Actually, this is not a deletion. It was in the wrong place in the file structure and it was moved to the right place. Line 282 in the file I sent is where it should be. It was moved there.

https://medium.com/superalgos/superalgos-translator-c76c6457b087
Translation work was done with this tool. I checked sentence by sentence and the Translator tool also corrects structural defects.

File: Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json (hunk 3 above)
Comment (devosonder) [reply Projects/Foundations/Schemas/Docs-Nodes/D/Data/Data-Tasks/data-tasks.json:98]: The last 2 comments you posted are just moving translations from the wrong place to the right place. Not an addition or deletion.

**Comment by devosonder** [comment]
> 

👍

**Comment by devosonder** [comment]
> Now 70 files reviewed 👍

Thank you for doing the Review so diligently and accurately. Great job. 🥇

File: Projects/Foundations/Schemas/Docs-Nodes/E/Exchange/Exchange-Learning-Products/exchange-learning-products.json [hunk 4]
```diff
@@ -9,18 +19,14 @@
...
-        "text": "Right click and select the pencil button to edit this tex. Replace it with a definition / summary. Hit ESC to exit edit mode.",
-        "translations": [
-            {
-                "language": "RU",
-                "text": "Щелкните правой кнопкой мыши и выберите кнопку с карандашом, чтобы отредактировать этот текст. Замените его на определение / резюме. Нажмите ESC, чтобы выйти из режима редактирования.",
-                "updated": 1645166371797
-            }
-        ]
-    }
-}
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Nodes/E/Exchange/Exchange-Learning-Products/exchange-learning-products.json:26]: I guess same thing here? Just moved to the correct position?

**Comment by gitforkgit** [comment]
111 files reviewed. Maybe I can merge this pull request tomorrow. :+1:

File: Projects/Foundations/Schemas/Docs-Nodes/E/Exchange/Exchange-Learning-Products/exchange-learning-products.json (hunk 4 above)
Comment (devosonder) [reply Projects/Foundations/Schemas/Docs-Nodes/E/Exchange/Exchange-Learning-Products/exchange-learning-products.json:26]: Yes. "type": Moved under "Exchange Learning Products". Where it should be.

**Comment by gitforkgit** [comment]
310 files I have looked through and they are good. The other files I have either not looked through yet or have found something where I am not sure if it is a small error. I will look at all the files I am not sure about tomorrow and report back if anything needs to be adjusted. The potential errors seem to be the same as other errors, so only a few things may need to be adjusted.
It also tells me that these 3 files have conflicts that need to be resolved: Projects/Foundations/Schemas/Docs-Nodes/E/Execution/Execution-Algorithm/execution-algorithm.json
Projects/Foundations/Schemas/Docs-Tutorials/T/Testing/Testing-a-Trading-Idea-The-Other-Stages/testing-a-trading-idea-the-other-stages-059-tutorial-step-is-there-room-for-nuance.json
This may be due to the fact that I have a different PR flagged. I will let you know tomorrow with the other feedback if and what should be changed. :+1:

**Comment by devosonder** [comment]
Conflicts are Resolved

**Comment by gitforkgit** [comment]
> Conflicts are Resolved

Awesome :+1:

File: Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json (hunk 1 above)
Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Nodes/C/Crypto/Crypto-Exchange/crypto-exchange.json:294]: Good that you sent me the article, I had not noticed anything about it.

File: Projects/Foundations/Schemas/Docs-Nodes/E/Execution/Execution-Algorithm/execution-algorithm.json [hunk 5]
```diff
@@ -131,7 +134,8 @@
                 {
                     "language": "TR",
                     "text": "Yürütme Algoritması Yapılandırması",
-                    "updated": 1654755849909
+                    "updated": 1654392450016,
+                    "style": "Title"
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Nodes/E/Execution/Execution-Algorithm/execution-algorithm.json:138]: Just out of interest did you translate this file in Superalgos itself and add the style "text" in the translation or did you use the automatic tool?

**Review by gitforkgit — APPROVED** [review]
Everything looks good, please answer the one question I asked today, this is not important to merge but can help to identify certain patterns to simplify reviewing. :+1:

File: Projects/Foundations/Schemas/Docs-Nodes/E/Execution/Execution-Algorithm/execution-algorithm.json (hunk 5 above)
Comment (devosonder) [reply Projects/Foundations/Schemas/Docs-Nodes/E/Execution/Execution-Algorithm/execution-algorithm.json:138]: I used the Superalgos Translator tool. And then I checked the translations myself. Finally, I created a PR.
